SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKDefaultWorkerJsonGenerator.cpp

```diff
@@ -60,6 +67,23 @@ bool GenerateAllDefaultWorkerJsons(bool& bOutRedeployRequired)
 					bAllJsonsGeneratedSuccessfully = false;
 				}
 			}
+			if (bFileExists && !Pair.Value)
+			{
+				UE_LOG(LogSpatialGDKDefaultWorkerJsonGenerator, Verbose, TEXT("Found worker json at %s"), *JsonPath);
+
+				IFileManager& FileManager = IFileManager::Get();
+				if (!FileManager.Delete(*JsonPath))
+				{
+					UE_LOG(LogSpatialGDKDefaultWorkerJsonGenerator, Verbose, TEXT("Failed to delete default worker json from %s"),
```

Comment:
Is this really a Verbose log? Under what conditions would this fail and how serious is it? I think at least upgrading it to Log is called for.

Reply:
Ah, good catch, I think I will make this a Warning. The generation one is an error, but this one does not hamper functionality, it is just potentially undesirable.

Reply:
Nevermind, will do error as well, it stops the calling function